Weekly highlights update (#2441)
giantswarm/docs: src/content/changes/highlights/2025-01-30.md

    - Introduced VPA support for CoreDNS to enhance scalability and performance.

- [Kong App](https://github.com/giantswarm/kong-app) version [4.5.0](https://github.com/giantswarm/kong-app/compare/v4.4.0...v4.5.0)
    - Updated Kong Ingress Controller for improved routing and security features.

- [Ingress NGINX App](https://github.com/giantswarm/ingress-nginx-app) version [3.9.4](https://github.com/giantswarm/ingress-nginx-app/compare/v3.9.3...v3.9.4)
    - Enhanced ingress management with updated images and synced upstream changes.

- [Vertical Pod Autoscaler App](https://github.com/giantswarm/vertical-pod-autoscaler-app) version [5.3.1](https://github.com/giantswarm/vertical-pod-autoscaler-app/compare/v5.3.0...v5.3.1)
    - Updated Helm release to improve resource management and scaling.

## Others

- [Object Storage Operator](https://github.com/giantswarm/object-storage-operator) version [0.10.0](https://github.com/giantswarm/object-storage-operator/compare/v0.9.0...v0.10.0)
    - Secured Azure storage with private access and updated policy management.

> - [n8n App](https://github.com/giantswarm/n8n-app) version [0.1.5](https://github.com/giantswarm/n8n-app/compare/v0.1.0...v0.1.5)
>     - Improved integration with updated application labeling.

Reviewer: ```suggestion
- [n8n App](https://github.com/giantswarm/n8n-app) version [0.1.5](https://github.com/giantswarm/n8n-app/compare/v0.1.0...v0.1.5)
    - A new AI workflow automation platform has been added.
```

- [Debug Toolbox](https://github.com/giantswarm/debug-toolbox) version [1.1.1](https://github.com/giantswarm/debug-toolbox/compare/v1.0.0...v1.1.1)
    - Released new tools for better debugging and troubleshooting.

- [Hello World App](https://github.com/giantswarm/hello-world-app) version [2.6.1](https://github.com/giantswarm/hello-world-app/compare/v2.6.0...v2.6.1)
    - Enhanced features with Gateway API support and updated documentation for easier usage.

## Docs

>     - [We have unified header and footer of our website](https://github.com/giantswarm/docs/pull/2445).
>     - [New guide on how to add your application logs to the managed Loki installation](https://github.com/giantswarm/docs/pull/2447).
>     - [We added the annotations you can tune in the Cluster AWS resources](https://github.com/giantswarm/docs/pull/2454).

Reviewer: ```suggestion
    - [We have unified header and footer of our website](https://github.com/giantswarm/docs/pull/2445).
    - [New guide on adding your application logs to the managed Loki installation](https://github.com/giantswarm/docs/pull/2447).
    - [We added the annotations you can tune in the Cluster AWS resources](https://github.com/giantswarm/docs/pull/2454).
```

    - Introduced support for customer-specific log tenancy and enhanced default configurations.

- [Cluster API Monitoring App](https://github.com/giantswarm/cluster-api-monitoring-app) version [1.16.1](https://github.com/giantswarm/cluster-api-monitoring-app/compare/v1.16.0...v1.16.1)
    - Improved security and compliance with updated RBAC permissions.

- [Prometheus Blackbox Exporter App](https://github.com/giantswarm/prometheus-blackbox-exporter-app) version [0.5.0](https://github.com/giantswarm/prometheus-blackbox-exporter-app/compare/v0.4.2...v0.5.0)
    - Enhanced security settings to meet compliance standards.

## Security

- [Kyverno Policy Operator](https://github.com/giantswarm/kyverno-policy-operator) version [0.1.1](https://github.com/giantswarm/kyverno-policy-operator/compare/v0.1.0...v0.2.0)
    - Enhanced CRD management and improved controller functionality for better policy enforcement.

- [Cert Manager App](https://github.com/giantswarm/cert-manager-app) version [3.9.0](https://github.com/giantswarm/cert-manager-app/compare/v3.8.2...v3.9.0)
    - Upgraded to the latest Cert-manager version for improved certificate management.

> - [Etcd Defrag App](https://github.com/giantswarm/etcd-defrag-app) version [0.1.1](https://github.com/giantswarm/etcd-defrag-app/compare/v0.1.0...v0.1.1)
>     - Improved configuration options for easier management of cluster endpoints.
> 

Reviewer: ```suggestion
```
